app/assets/javascripts/modules/moj.emailPreview.js

```diff
@@ -13,7 +13,20 @@
     },
 
     handleClick: function(event) {
-      event.target.search = $(this.el).serialize();
+      event.preventDefault();
+
+      var ajaxOptions = {
+        url:    event.target.href,
+        method: 'POST',
```

Comment:
It actually is a 'PUT', the serialised  form has a hidden input '_method=put' so it is perfectly fine. And not all browsers (yes internet explorer, i'm looking at you!) supports does PUT and DELETE requests. Which is why rails always add the ```_method``` in forms.